dir.props

```diff
@@ -114,7 +114,7 @@
   <!-- Dependency auto-upgrade configuration -->
   <ItemGroup>
     <DependencyBuildInfo Include="CoreFx">
-      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/$(DependencyBranch)</RawUrl>
+      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/dev/api</RawUrl>
```

Comment:
Hmm, I thought about my suggestion a bit more and I'm pretty sure it'll still result in merge conflicts on the CurrentRef/CurrentBranch... they would be localized and easy to resolve, but that's not clean.

A way I can think of to get rid of those is to actually use a different file for each branch's dependencies, leaving branch dependency info checked in for any "upstream" branch. (E.g. dev/api would keep master's dependencies file checked in. Once dev/api is merged back into master, dev/api's deps file could be cleaned up whenever convenient.) An auto-upgraded property changed to the value of `$(CurrentBranch)` during the last auto-PR would point to the file to use. No merge conflict there because master's value never changes until dev/api is merged back in.

(They don't need to be multiple files--they could be differently-named properties in a single expanding file.)

That seems pretty complex to "just" avoid merge conflicts, but I'm not sure how valuable that is.